examples/inspection/plot_linear_model_coefficient_interpretation.py

```diff
@@ -632,14 +646,25 @@
 # A Lasso model identifies the correlation between
 # AGE and EXPERIENCE and suppresses one of them for the sake of the prediction.
 #
+# It is important to keep in mind that the coefficients that have been
+# dropped may still be related to the outcome in by themselves: the model
+# chose to suppress them because they brings little or no additional
+# information on top of the other features. Additionnaly, this selection
+# is unstable for correlated features, and should be interpreted with
+# caution.
+#
 # Lessons learned
 # ---------------
 #
-# * Feature importance could be extrapolated from the coefficients only after
-#   having scaled them to the same unit of measure.
-# * Coefficients in multiple linear models represent conditional dependencies
-#   between a given feature and the target.
-# * Correlated features induce variability in the coefficients of linear
-#   models.
+# * Coefficients must be scaled to the same unit of measure to retrieve
+#   feature importance. Scaling them with the standard-deviation of the
+#   feature is a useful proxy.
+# * Coefficients in multivariate linear models represent the dependency
+#   between a given feature and the target, **conditional** on the other
+#   features.
+# * Correlated features induce instabilities in the coefficients of linear
+#   models and there effects cannot be well teased appart.
 # * Different linear models respond differently to feature correlation and
 #   coefficients could significantly vary from one another.
+# * Inspecting coefficients across the folds of a cross-validation loop
+#   gives an idea of their stability.
```

Comment:
A question about this and all other recurrences of "stability" and derived words about coefficients.
Using it worthed me a [comment](https://github.com/scikit-learn/scikit-learn/pull/15706#discussion_r353219620), so I tried to avoid it... and now it's back... :) is there really a "truth" to look for? Thanks!

Reply:
I'd say that's fine here

I suggested variance / variability because that's really what it's about, and it's important for section titles to be accurate. But stability in this context isn't ambiguous.